apps/vanilla/src/pages/exercise1/styles/colors.css

```
@@ -0,0 +1,9 @@
+:root {
+  --primary-color: #f15924;
+  --on-primary-color: #fff;
```

Just for consistency  🌚
```suggestion
  --on-primary-color: #ffffff;
```

---

@gevorg-gevorgyan-saritasa Stylelint warn about it and require to simplify

---

Also in wiki its'same with stylent warning https://wiki.saritasa.rocks/frontend/languages/css/#dont-be-too-precise:~:text=dark%2Dcolor%3A-,%23000,-%3B%0A%20%20%2D%2Dprimary%2Dlight